tests/components/orderentry/OrdersTable.test.jsx

```diff
@@ -123,3 +181,33 @@ describe('Orders component test-suite', () => {
     expect(props.dispatch).toBeCalled();
   });
 });
+
+describe('getUUID() method', () => {
+  it('should call getUUID()', () => {
+    const renderedComponent = getComponent().instance();
+    sinon.spy(renderedComponent, 'getUUID');
+    renderedComponent.getUUID(items, itemName);
+    expect(renderedComponent.getUUID.calledOnce).toEqual(true);
+  });
```

Comment:
I see here that the `getUUID` method is called when the discontinue method is called, I'd advise you assert for this when the `disContinue` method is triggered instead of just calling the `getUUID` method directly and asserting that it was called.

currently, this test-suite doesn't give an idea of how the `getUUID` method relates to the functionality of the component in anyway.

Reply:
@Efosaok  it's a method in the component so I don't get how it does not relate to the functionality of the component in any way

Reply:
I agree it is a method, but how does this method affect the component, your test doesn't seem to show how, and I realised it is usually called as a helper function in another method, so in a test-suite that involves the method that calls `this.getUUID()`, assert that it was called, that way, it would make more sense and give meaning to the function of the method.

Reply:
@Efosaok is there a rule against writing tests for helpers ? and If I can remember correctly there is an exact method on this code base with tests also check here, https://github.com/openmrs/openmrs-owa-orderentry/blob/master/app/js/components/drugOrderEntry/addForm/DraftDataTable.jsx#L22
tests here 
https://github.com/openmrs/openmrs-owa-orderentry/blob/master/tests/components/DrugOrderEntry/addForm/DraftDataTable.test.jsx#L109